definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "01/06/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseEventID": "FR_generalLetter",
```

Comment:
would CCD not allow us to have all of the below changes in prod files and this change be the only one in a non-prod file so no user can actually submit the event?